scrapy/meta.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: scrapy
+  version: "1.0.3"
```

Comment:
@dangra in that case, we can increase the build number to add a new (updated) package for the same version and it will be picked by conda automatically. For example, current build is `py27_0` and next build for the same version will be `py27_1`.